cryptopals/set1/4/4.txt

```diff
@@ -0,0 +1,327 @@
+0e3647e8592d35514a081243582536ed3de6734059001e3f535ce6271032
```

Comment:
I'd actually say, keep it. The reason is that then you have a consistent thing, and you don't need to depend on an external file. Additionally, it is a tiny file compared to how much GH allows us to keep, so I'd say keep it :)